Programs/LargestnumberFind.c

```diff
@@ -0,0 +1,244 @@
+ #include<stdio.h>
+ #include<conio.h>
+ main()
+ {
+      int a,b,c,d,e,f;
+      clrscr();
```

Comment:
clrscr() is from old C compilers. It does not work with new C Compilers. 
Use system("cls") from the header "stdlib.h" instead.